tests/autopilot/autopilot_test.go

```diff
@@ -252,7 +251,7 @@ var _ = Describe(fmt.Sprintf("{%sRestartAutopilot}", testSuiteName), func() {
 						&appsapi.Deployment{
 							ObjectMeta: meta_v1.ObjectMeta{
 								Name:      autDeploymentName,
-								Namespace: autDeploymentNamespace,
+								Namespace: Inst().DriverNamespace,
```

Comment:
we can't use Inst().DriverNamespace here as portworx and autopilot could be in different namespaces